docs/content/direct/example-wecs.md

```diff
@@ -7,13 +7,13 @@ register them with the hub as descibed in the
 1. Execute the following commands to create two kind clusters, named `cluster1` and `cluster2`, and register them with the OCM hub. These clusters will serve as workload clusters. If you have previously executed these commands, you might already have contexts named `cluster1` and `cluster2`. If so, you can remove these contexts using the commands `kubectl config delete-context cluster1` and `kubectl config delete-context cluster2`.
 
    ```shell
-   : set flags to "" if you have installed KubeStellar on an OpenShift cluster
-   flags="--force-internal-endpoint-lookup"
+   : set flags to "--singleton" if you have installed KubeStellar on an OpenShift cluster
+   flags="--singleton --force-internal-endpoint-lookup"
    clusters=(cluster1 cluster2);
    for cluster in "${clusters[@]}"; do
       kind create cluster --name ${cluster}
       kubectl config rename-context kind-${cluster} ${cluster}
-      clusteradm --context imbs1 get token | grep '^clusteradm join' | sed "s/<cluster_name>/${cluster}/" | awk '{print $0 " --context '${cluster}' '${flags}'"}' | sh
+      clusteradm --context imbs1 get token | grep '^clusteradm join' | sed "s/<cluster_name>/${cluster}/" | awk -v fl="$flags" '{print $0 " --context '${cluster}' " fl}' | sh
```

Comment:
Yikes the injection of `$flags` is getting complicated. Couldn't we do it more simply, as follows?

```shell
      clusteradm --context imbs1 get token | grep '^clusteradm join' | sed "s/<cluster_name>/${cluster}/" | awk '{print $0 " --context ${cluster} ${flags}"}' | sh
```

Also: does this PR really need a change here?

Reply:
It looks like `awk` have an issue with environment variables so when I added the second flag it didn't work. The `-v` makes the variable native to the `awk` so this is the recommended way.
I tried to reverse the quotes but this didn't work either.
We already have `$flags` and it's in the user-guide, and `--singleton` is a flag (just like `--force-internal-endpoint-lookup`) so it felt natural to add it to `$flags`

Reply:
The original version of this command is not even asking `awk` to deal with the shell variable `flags` --- the syntax was asking the shell to interpolate the value of `flags` into the command line.

Reply:
I think we do not have to be too rigid about the word "flags". This is a variable that we created to solve a problem. We do not have to make the solution more complicated than necessary. Also, see that `--context` flag that is already not in `flags`? I think the simplest solution is to make `--singleton` just as constant and direct and literal as `--context`. Rename the variable if you like.